assets/app/actions/siteActions.js

```diff
@@ -65,7 +66,15 @@ export default {
 
   fetchSiteConfigsAndAssets(site) {
     return this.fetchSiteConfigs(site).then((site) => {
-      return this.fetchSiteAssets(site);
+      return this.fetchSiteAssets(site).then((site) => {
```

Comment:
I think that just the navigation file might be necessary to view the tree. And not even the file, just if it is there or not.

Ya, I'm not totally opposed to the idea of forcing repos to have a `navigation.json`-like thing (and our templates should definitely have them) but I also like the idea that any Jekyll site be added to Federalist and they can configure it themselves to take advantage of features. And I agree that its out of scope of this PR :)